iml-agent/src/action_plugins/check_stratagem.rs

```diff
@@ -0,0 +1,10 @@
+use crate::{agent_error::ImlAgentError, cmd::cmd_output};
+
+pub async fn check_stratagem(_: ()) -> Result<bool, ImlAgentError> {
+    let rpm = cmd_output("rpm", vec!["--query", "rust-iml-stratagem"]).await?;
```

Comment:
```suggestion
    let rpm = cmd_output("rpm", vec!["--query", "lipe"]).await?;
```

fwiw, the actual package name we want to query here is called `lipe`.

Reply:
I was under impression it's one of our packages and it's ... not?

Reply:
Correct.

Reply:
Fixed.